CompanyEmployees.Application/Commands/CheckInAttendanceCommand.cs

```diff
@@ -0,0 +1,6 @@
+using MediatR;
+using Shared.DataTransferObjects;
+
+namespace CompanyEmployees.Application.Commands;
+
+public sealed record CheckInAttendanceCommand(AttendanceForCreationDto Attendance) : IRequest<AttendanceDto>;
```

Comment:
The parameter name 'Attendance' is PascalCase, which is inconsistent with C# naming conventions for method or constructor parameters. Parameters should use camelCase to distinguish them from types and properties.

```suggestion
public sealed record CheckInAttendanceCommand(AttendanceForCreationDto attendance) : IRequest<AttendanceDto>;
```